packages/requisitions/src/ResponseRequisition/api/hooks.ts

```diff
@@ -18,8 +18,10 @@ import {
   useNotification,
   useTranslation,
   Column,
+  RegexUtils,
 } from '@openmsupply-client/common';
 import { getResponseQueries, ListParams } from './api';
+import { useItemFilter } from '../../RequestRequisition/api';
```

Comment:
Not sure if this is okay. The Request and Response requisition modules seem quite well-separated, so is it bad to share a common filter string?

Reply:
can't see an issue with it - and it's nice to be using the same hook from different components